### DISCUSSION TITLE
VN/VP ports from system config causing critical warning

### DISCUSSION THREAD

**Discussion by wnew** [Xilinx/open-nic-shell#47] [Q&A]
The vivado IP system_management wizard  seems to have enable VP/VN set to true. This causes critical errors, at least on the AU50 as the ports are not brought up to the top level. Would setting this to false break other boards?

I can submit a pull request if required.

**Comment by cneely-amd** [comment]
The short answer is that I'm not sure.  I can see VP / VN described here: https://www.xilinx.com/support/documents/user_guides/ug580-ultrascale-sysmon.pdf , e.g. pg. 21.  
If you submit a pull request, I can test it on the U250, which is the board that I currently have available.
--Chris

**Reply by wnew** [reply]
Thanks Chris.

I have issued a pull request. Please let me know if you would like any further information on the issue. Looking forward to hearing your results.

BTW I am building on Vivado 2023.1
